### DISCUSSION TITLE
-nW option doesnt work

### DISCUSSION THREAD

**Discussion by Anthonymcqueen21** [projectdiscovery/subfinder#731] [Ideas]
Hello everyone, my problem is the -nW option, i use it according to the user's manual, and it does not work. I am trying to use it like this for example.

**" ./subfinder -d domain.com -nW = Found 0 subdomains found"

" ./subfinder -nW domain.com -d = Found 0 subdomains found"**

Am i using this tool wrong or inputting the commands incorrectly because all I want is VALID subdomains that are active and not dead ones which waste's time to get through any help would be greatly appreciated, thank you.

**Comment by forgedhallpass** [comment]
I've just tried it with the latest version, without API keys (default setting) and it works.

Maybe you have some connection issues? Have you tried using the `-v` for verbose output?

```sh
 subfinder -d domain.com -nW


               __    _____           __         
   _______  __/ /_  / __(_)___  ____/ /__  _____
  / ___/ / / / __ \/ /_/ / __ \/ __  / _ \/ ___/
 (__  ) /_/ / /_/ / __/ / / / / /_/ /  __/ /    
/____/\__,_/_.___/_/ /_/_/ /_/\__,_/\___/_/ v2.5.5

		projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
By using subfinder, you also agree to the terms of the APIs used.

[INF] Loading provider config from '~/.config/subfinder/provider-config.yaml'
[INF] Enumerating subdomains for 'domain.com'
bilper29x.domain.com
tmconsul.domain.com
idlehands.domain.com
goldstoradia.domain.com
...
[INF] Found 590 subdomains for 'domain.com' in 1 minute 6 seconds
```

**Reply by Anthonymcqueen21** [reply]
Thank you i will give it a shot and hope it works for my target.
